server/pom.xml

```diff
@@ -205,6 +205,8 @@
                                     <extraArgument>-verbose:gc</extraArgument>
                                     <!--netty-->
                                     <extraArgument>-Dio.netty.leakDetectionLevel=advanced</extraArgument>
+                                    <!--run by *.bat or *.sh-->
+                                    <extraArgument>-Drun.by-startup=true</extraArgument>
```

Comment:
wins and unix both contain `-Drun.by-startup=true`. How distinguish between them?

Reply:
I renamed the jvm argument name. PTAL.